packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -46,10 +46,12 @@ function Tabs(props: Props): JSX.Element {
     }
   }
 
-  const handleTabChange = (event) => {
-    const selectedTab = event.target;
+  const handleTabChange = (
+    event: React.FocusEvent<HTMLLIElement> | React.MouseEvent<HTMLLIElement>,
+  ) => {
+    const selectedTab = event.currentTarget;
```

Comment:
we actually wants to use currentTarget here, instead of target in case if header is not going to be simple element 

https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget